### DISCUSSION TITLE
Defensebot ammo OnImpact doesnt show particles

### DISCUSSION THREAD

**Discussion by DefaultStringOne** [FakeFishGames/Barotrauma#15440] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

< Projectile hitscan="true">
      < StatusEffect type="OnImpact" target="UseTarget">
              < Explosion range="150.0" showeffects="false" >
      </ StatusEffect>
</ Projectile>

This code block does not work when Projectile Item is spawned via creature ranged attack and hits another creature.
If it hits wall, Status effect executes, particles, explosion, damage etc. But once creature limb is hit, nothing.

Added video demonstrating with defensebot and nucleargunbolt.

https://github.com/user-attachments/assets/fc249b22-0196-480d-bd73-0f1ffc034fb6
https://github.com/user-attachments/assets/86a135fa-d975-45ec-a194-74e747f754e5

### Reproduction steps

Spawn enemy defensebot and watch closely explosion thats supposed to appear on impact when hitting creature limb, for testing purposes add some more particles to status effect if its hard to see.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
The issue has been addressed in our private repo now, fix coming up in the next unstable build and the upcoming update! :)

**Comment by DefaultStringOne** [comment]
U are best! Now can finish mod i had in limbo for months. +10
